server.js

```diff
@@ -2,6 +2,7 @@ const express = require('express');
 const ws = require('ws');
 const app = express();
 const port = process.env.PORT || 3000;
+const host = process.env.HOST || '127.0.0.1';
```

Comment:
in what scenario is that useful? when you are running those pages on some external test server?

Reply:
Yes, that, or a VM. I used it to bind to `0.0.0.0` in WSL.